sport/test/football/feed/CompetitionsTest.scala

```diff
@@ -147,15 +166,15 @@ class CompetitionsTest extends FreeSpec with Matchers with OptionValues {
         awayTeam = team.copy(id = awayName, name = awayName, score = Some(awayScore)),
       )
 
-    private def fixture(homeName: String, awayName: String, date: ZonedDateTime) =
+    def fixture(homeName: String, awayName: String, date: ZonedDateTime) =
       _fixture.copy(
         id = s"fixture $homeName $awayName ${date.format(DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ssZ"))}",
         date = date,
         homeTeam = team.copy(id = homeName, name = homeName, score = None),
         awayTeam = team.copy(id = awayName, name = awayName, score = None),
       )
 
-    private def result(
+    def result(
         homeName: String,
         awayName: String,
         homeScore: Int,
```

Comment:
I've exposed these three methods so that I can use them to construct a sequence of matches in the test for greater visibility over what matches we are testing against.